### ISSUE TITLE
How can I integrate ResponseCard feature of LEX in my android app?

### ISSUE THREAD

**Issue by PikkaPikkachu** [aws/aws-sdk-android#386]
### To help us solve your problem better, please answer the following list of questions.

* What service are you using?
Amazon Lex
* In what version of SDK are you facing the problem?
The latest one
* Is the issue limited to Simulators / Actual Devices?
Both
* Can your problem be resolved if you bump to a higher version of SDK?
No, I don't think so
* Is this problem related to specific Android/OS version?
No
* Can you give us steps to reproduce with a minimal, complete, and verifiable example? Please include any specific network conditions that might be required to reproduce the problem.
None
* Please include a stacktrace if applicable.
None

I just need to know, how I can integrate the response card feature in my android app. I see no function in the Response.java which can tell me whether a cardResponse has been sent back by the bot or not?
Please guide me through this!
Thanks for help!

**Comment by kvasukib** [comment]
Hi @PikkaPikkachu, Thanks for reporting to us. What's the use-case that you are trying to solve with Lex? What's the requirement for finding whether a cardResponse has been sent back by the bot or not?

**Comment by PikkaPikkachu** [comment]
Hi @kvasukib, 
I figured that that the mobileconnectors/lex/interactionkit is based on PostContent request, so I ended up writing my own library for PostText request which returns a JSON response with ResponseCard attribute and that worked well for me. 

Anyone else who faces the same issue, here is the link to the library: https://github.com/PikkaPikkachu/Post-Text-Request-Lex

I am closing this issue now. 
Thanks for your reply! 

Regards,
PikkaPikkachu
